p1.py

```diff
@@ -1,4 +1,4 @@
-#this works
+#start
 #https://leetcode.com/problems/two-sum/
 #click the above link to see the problem definition
 class Solution:

```

Comment:
The time complexity of this solution is : O(n2) 

You can do it in O(n) time with O(n) space using a Hash Map.